apps/ui/src/components/TransactionsListItem.vue

```diff
@@ -53,7 +52,7 @@ const parsedTitle = computedAsync(
       <slot name="left" />
       <IH-cash v-if="tx._type === 'sendToken'" />
       <IH-photograph v-else-if="tx._type === 'sendNft'" />
-      <ICStake v-else-if="tx._type === 'stakeToken'" />
+      <IC-Stake v-else-if="tx._type === 'stakeToken'" />
```

Comment:
It should be
```<IC-stake v-else-if="tx._type === 'stakeToken'" />```

Reply:
looks like you are looking at outdated change

Reply:
Ok my bad nvm